Loop/Models/ServiceAuthentication/ShareService.swift

```diff
@@ -36,14 +38,23 @@ class ShareService: ServiceAuthentication {
                 value: url?.absoluteString,
                 options: [
                     (title: NSLocalizedString("US", comment: "U.S. share server option title"),
-                     value: DexcomShareURL.absoluteString)
+                     value: DexcomShareURL.absoluteString),
+                    (title: NSLocalizedString("Outside US", comment: "Outside US share server option title"),
+                     value: KnownShareServers.NON_US.rawValue)
+
                 ]
             )
         ]
 
+        if customServer.characters.count > 0 {
```

Comment:
Or maybe don't even set a string at all in the no-custom-server case; just have a block of code that is completely commented out.